### DISCUSSION TITLE
bg image in reactjs giving error

### DISCUSSION THREAD

**Discussion by 18121A05L2** [tailwindlabs/tailwindcss#9374] [Help] [answered]
hello mates , i am using react js 

-->for this tailwind css ,  bg-[url('/home/ratinginset.png')]

--> error 

Compiled with problems:X

ERROR in ./src/index.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[2]!./node_modules/source-map-loader/dist/cjs.js!./src/index.css) 5:36-85

Module not found: Error: Can't resolve '/home/ratinginset.png' in 'D:\my-project\src'

--> actually this was worked fine in next js , and i am now moving code to react js , all normal images are working but not background 

--> can anyone help me why i am getting this error ?

**Comment by wongjn** [comment, marked as answer]
As the error states:

> Module not found: Error: Can't resolve '/home/ratinginset.png' in 'D:\my-project\src'

CSS loader has failed to find the file for the background image you have given, `/home/ratinginset.png`. I would presume that you would not want an absolute file path here. You should look at using a relative path, relative to the input CSS file used by Tailwind.

**Reply by 18121A05L2** [reply]
bro , but that image is present in the home section and when I use for normal image of src="/home/ratinginset.png" it is working  and i havent got that thing how to use  a relative path, relative to the input CSS file used by Tailwind ? 

--> can u elaborate a bit bro or send any examples or links

**Reply by wongjn** [reply]
When used as an `<img>` `src` attribute, `/home/ratinginset.png` will resolve to the root URL (designated by the starting `/`) by the **browser**. However, when using it in a Tailwind class, `/home/ratinginset.png` will resolve to the root of the **machine**, due to it being resolved at build time on said machine. As per [the CSS Loader documentation](https://webpack.js.org/loaders/css-loader):

> The `css-loader` interprets `@import` and `url()` like `import`/`require()` and will resolve them.

**Reply by 18121A05L2** [reply]
thanks bruh
